### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by yelllowsin** [harmony-one/auto-node#37]
There is an installation error happening on current version installed following instructions [here](https://docs.harmony.one/home/validators/node-setup/installing-updating/installing-node/using-autonode/install-and-run). More than one user reported me this issue and is happening for me as well. Can you please take a look @Daniel-VDM 

```
Installing collected packages: pycparser, cffi, cryptography, requests, ptyprocess, pexpect, pyhmy, AutoNode
Successfully installed AutoNode-1.0.3 cffi-1.14.3 cryptography-2.9.2 pexpect-4.8.0 ptyprocess-0.6.0 pycparser-2.20 pyhmy-20.5.15 requests-2.23.0
[AutoNode] Initilizing python3 library
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/harmony/.local/lib/python3.6/site-packages/AutoNode/__init__.py", line 90, in <module>
    _init()
  File "/home/harmony/.local/lib/python3.6/site-packages/AutoNode/__init__.py", line 71, in _init
    cli.environment.update(cli.download(cli_bin_path, replace=False, verbose=False))
  File "/home/harmony/.local/lib/python3.6/site-packages/pyhmy/cli.py", line 377, in download
    elif os.path.exists(f"{get_gopath()}/src/github.com/harmony-one/bls") \
  File "/home/harmony/.local/lib/python3.6/site-packages/pyhmy/util.py", line 42, in get_gopath
    return subprocess.check_output(["go", "env", "GOPATH"]).decode().strip()
  File "/usr/lib/python3.6/subprocess.py", line 356, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.6/subprocess.py", line 423, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'go': 'go'
```
<img width="841" alt="Screenshot at Sep 24 14-02-37" src="https://user-images.githubusercontent.com/7217595/94176979-40c99880-fe6f-11ea-9c3e-42f6f53c65a7.png">

Tested on:

**Freshly Linux VPS (Ubuntu 18.04 LTS)**

**Comment by Daniel-VDM** [comment]
This has been fixed with version `1.0.4`.

Tests on a fresh machine with Ubuntu 20.04 LTS

The underlying issue was that `hmy` (CLI) for Linux is no longer released as a dynamically linked binary (as it is with MacOS). This was an assumption made by the underlying `pyhmy` lib. I've updated the lib & auto-node to consume said lib.

Users can just re-run the first time install steps and it should work. Some people may have cached links and might have to re-run the install script multiple times if they see an error.
